content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
-
-If you do not have a degree, you can do an undergraduate degree which leads to QTS.
-
-Different types of training have different eligibility criteria, [funding options](/funding-your-training) and qualifications. Lots of courses also allow you to study part-time.
 
-You can also [get support to improve your subject knowledge](/guidance/become-a-teacher-in-england#subject-knowledge-enhancement-courses) to supplement your training.
-
-Not looking for primary or secondary training? Consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
-
-$get-a-tta$
-
-$pgce$
-
-PGCE programmes combine academic study with a minimum of 24 weeks spent training in schools.
-
-You do not need an academic qualification such as a PGCE to teach in England, but it may be useful if you want to teach in another country.
-
-Many PGCE courses give you credits that count towards a master’s degree.
+---
 
-You can also [get funding](/funding-your-training).
+## Train to get qualified teacher status
 
-#### Postgraduate Diploma in Education
+You need ‘qualified teacher status’ to work in the majority of schools in England including state maintained primary, secondary and special schools.
 
-Some providers offer a Postgraduate Diploma in Education (PGDE) instead of a PGCE. A PGDE sometimes offers more masters credits than a PGCE.
+There are different ways to get qualified teacher status, for example:
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a PGCE or PGDE <span>course</span></a>
-</p>
+* [if you want to go to university as a postgraduate student](#go-to-university-as-a-postgraduate-student)
 
-$school-direct-fee-funded$
+* [if you want to learn on the job in schools near you](#learn-on-the-job-in-schools-near-you)
 
-You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+* [if you do not have a degree](#if-you-do-not-have-a-degree)
 
-You’ll be based in a school but you will not be employed by them.
+* [if you already have teaching experience](#if-you-already-have-teaching-experience)
 
-You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+You can also find specialised programmes [if you're a career changer](#if-youre-a-career-changer) or [if you have a doctorate](#if-you-have-a-doctorate).
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+Not looking for primary or secondary teacher training? Consider teaching [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
 
-You can also [get funding](/funding-your-training).
+### Go to university as a postgraduate student
```

Comment:
Wondering if saying ‘Return’ makes it a bit clearer that this is a postgraduate route?

```suggestion
### Return to university as a postgraduate student
```

Reply:
Oh I like it